ocs_ci/ocs/ocp.py

```diff
@@ -681,6 +681,10 @@ def wait_for_delete(self, resource_name="", timeout=60, sleep=3):
             bool: True in case resource deletion is successful
 
         """
+        if config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM:
+            timeout = 720
+        else:
+            timeout = 60
```

Comment:
this overwrite the timeout if user want to pass timeout as parameter

Reply:
I will remove the else condition so that it won't affect other platforms

Reply:
For IBM Power platform resources are not getting deleted in 60 sec . We are getting many timeout failures due to resources like PVCs not getting deleted in mentioned timeout value.

Reply:
@vasukulkarni ^^

Reply:
@vavuthu ^^